<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEmbONG</class>
 <widget class="QDialog" name="DialogEmbONG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de Lineas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NÃºmero fijo    +504 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contacto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Puesto de contacto: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>140</red>
         <green>156</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>81</red>
         <green>105</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>27</green>
         <blue>124</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>14</red>
         <green>36</green>
         <blue>165</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>154</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>140</red>
         <green>156</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>81</red>
         <green>105</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>27</green>
         <blue>124</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>14</red>
         <green>36</green>
         <blue>165</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>154</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>27</green>
         <blue>124</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>140</red>
         <green>156</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>81</red>
         <green>105</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>27</green>
         <blue>124</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>14</red>
         <green>36</green>
         <blue>165</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>27</green>
         <blue>124</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>27</green>
         <blue>124</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>248</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_adress">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_contact">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_tel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_job">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_email">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="num_lineas">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nuevo Cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Beneficiarios /  Tipo </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_car">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="desc">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Descuento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>164</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Cliente</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_clientes">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>23</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>39</red>
         <green>58</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>15</red>
         <green>26</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>47</green>
         <blue>234</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>23</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>39</red>
         <green>58</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>15</red>
         <green>26</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>47</green>
         <blue>234</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>23</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>39</red>
         <green>58</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>24</red>
         <green>47</green>
         <blue>234</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Empresa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Embajada / ONG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Particular Prepago</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Particular Postpago</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha de Corte</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pago">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Factura de consumo   $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_fac_c">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_fecha">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ultimo pago</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>FacturaciÃ³n Mensual  $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Empresa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Entreprenuer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Elite</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Enterprise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prime</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="le_saldo">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo Incial       Lps.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cliente Corporativo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cliente Particular</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="fac_m">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>label_21</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>le_name</zorder>
  <zorder>le_adress</zorder>
  <zorder>le_contact</zorder>
  <zorder>le_tel</zorder>
  <zorder>le_job</zorder>
  <zorder>le_email</zorder>
  <zorder>num_lineas</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>le_car</zorder>
  <zorder>pushButton</zorder>
  <zorder>desc</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>cb_clientes</zorder>
  <zorder>label_13</zorder>
  <zorder>le_pago</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>le_fac_c</zorder>
  <zorder>le_fecha</zorder>
  <zorder>le_id</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>cb2</zorder>
  <zorder>le_saldo</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>line</zorder>
  <zorder>fac_m</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
